testdata/baselines/reference/submoduleAccepted/conformance/emitModuleCommonJS(module=commonjs).errors.txt.diff

```diff
@@ -16,10 +15,4 @@
 +!!! error TS2580: Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
```

Comment:
This error is still here because `IsRequireCall` is missing the `requireStringLiteralLikeArgument` arg, which would normally let callers decide if the argument is even checked.